x-pack/plugins/security_solution/public/common/lib/kibana/hooks.ts

```diff
@@ -157,8 +157,8 @@ export const useGetUserCasesPermissions = () => {
 
   useEffect(() => {
     setCasesPermissions({
-      crud: !!uiCapabilities[CASES_FEATURE_ID].crud_cases,
-      read: !!uiCapabilities[CASES_FEATURE_ID].read_cases,
+      crud: !!uiCapabilities[CASES_FEATURE_ID]?.crud_cases ?? false,
```

Comment:
I don't think that `?? false` is needed. The `!!` will convert `undefined` to `false`.

Reply:
right! overkill defensive programming :D